cmake/util/thrift.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Meta
```

Comment:
shouldn't this be part of the module itself, either in modules/ or in the module itself?

Reply:
Sure - I can move it there